io/pci/distro_tools.py.data/distro_tools.yaml

```diff
@@ -1,4 +1,5 @@
 pci_device: ""
+adapter_type: "net"
```

Comment:
Why only net here ?

Reply:
just for default i gave.

Reply:
This is not needed if you use get_class_name()